developer-workflow/sbom.rst

```diff
@@ -0,0 +1,45 @@
+Software Bill-of-Materials (SBOM)
+=================================
+
+Software Bill-of-Materials (abbreviated as "SBOM") is a document for sharing information about
+software and how it's been composed. This format is used most often in the security space for
+checking software and its dependencies for vulnerabilities using vulnerability databases like
+`CVE <https://www.cve.org/>`_ and `OSV <https://osv.dev/>`_. The SBOM format that the CPython project uses is `SPDX <https://spdx.github.io/spdx-spec/v2.3/>`_
+which can be transformed into other formats if necessary by consumers.
+
+There are multiple sources of third-party dependencies for CPython.
+Some are vendored into the source code of CPython itself (like ``mpdecimal`` vendored at ``Modules/_decimal/libmpdec``)
+or they could be optionally pulled in during builds like Windows using dependencies from the `python/cpython-source-deps <https://github.com/python/cpython-source-deps>`_ repository.
+
+Whenever adding or updating a third-party dependency, an update will likely need to be done
+to the SBOM in order to track the version and software identifiers.
+
+Updating a dependency
+---------------------
+
+The SBOM for CPython's bundled dependencies is kept at ``Misc/sbom.spdx.json``. Instead of updating this document
+directly, there is a tool for automatically updating the document at ``Tools/build/generate_sbom.py``.
```

Comment:
```suggestion
The SBOM for CPython's bundled dependencies is kept at
:cpy-file:`Misc/sbom.spdx.json`. Instead of updating this document directly,
there is a tool for automatically updating the document at
:cpy-file:`Tools/build/generate_sbom.py`.
```